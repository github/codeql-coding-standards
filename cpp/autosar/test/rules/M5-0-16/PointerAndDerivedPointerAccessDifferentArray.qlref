rules/M5-0-16/PointerAndDerivedPointerAccessDifferentArray.ql