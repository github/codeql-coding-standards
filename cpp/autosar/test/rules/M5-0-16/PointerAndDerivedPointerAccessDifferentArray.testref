cpp/common/test/rules/donotusepointerarithmetictoaddressdifferentarrays/DoNotUsePointerArithmeticToAddressDifferentArrays.ql