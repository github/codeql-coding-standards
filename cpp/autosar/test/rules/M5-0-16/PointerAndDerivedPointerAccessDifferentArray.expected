| test.cpp:4:13:4:18 | ... + ... | Array pointer p2 points 1 element passed the end of $@. | test.cpp:2:7:2:8 | l1 | l1 |
| test.cpp:5:13:5:18 | ... + ... | Array pointer p3 points 1 element passed the end of $@. | test.cpp:2:7:2:8 | l1 | l1 |
| test.cpp:6:13:6:18 | & ... | Array pointer p4 points 1 element passed the end of $@. | test.cpp:2:7:2:8 | l1 | l1 |
| test.cpp:11:8:11:11 | ... -- | Array pointer p7 points 1 element passed the end of $@. | test.cpp:2:7:2:8 | l1 | l1 |
| test.cpp:12:8:12:9 | p3 | Array pointer p8 points 1 element passed the end of $@. | test.cpp:2:7:2:8 | l1 | l1 |
